addOns/ascanrules/src/main/java/org/zaproxy/zap/extension/ascanrules/CodeInjectionScanRule.java

```diff
@@ -289,4 +291,41 @@ private boolean testAspInjection(String paramName) {
     private static int getRandomValue() {
         return RAND.nextInt(MAX_VALUE) + 1;
     }
+
+    private AlertBuilder createAlert(
+            String name, String param, String attack, String evidence, String ref) {
+        return newAlert()
+                .setConfidence(Alert.CONFIDENCE_MEDIUM)
+                .setName(name)
+                .setParam(param)
+                .setAttack(attack)
+                .setEvidence(evidence)
+                .setAlertRef(String.valueOf(getId()) + ref);
+    }
+
+    @Override
+    public List<Alert> getExampleAlerts() {
+        List<Alert> alerts = new ArrayList<>();
+        alerts.add(
+                createAlert(
+                                Constant.messages.getString(MESSAGE_PREFIX + "name.php"),
+                                "param",
+                                PHP_PAYLOADS[0],
+                                PHP_CONTROL_TOKEN,
+                                "-1")
```

Comment:
This should be extracted to a method to avoid duplicating the name and the reference.